mobile/src/pages/Login.tsx

```diff
@@ -0,0 +1,54 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { useState } from 'react';
+import { Button, TextInput, Text, Pressable, View } from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  return (
+    <View className={'flex-1 w-full'}>
+      <View className={'w-full flex-row place-content-start mt-16 ml-2'}>
+        <Pressable>
+          <Icon size={45} name="arrow-back-outline" />
+        </Pressable>
+      </View>
+      <View className={"flex-1 flex ml-8 mt-8"}>
+
+        <Text className={"font-bold mb-3 text-xl"}>Enter your username:</Text>
+
+        <View className={"rounded-lg bg-slate-300 w-80 h-12 mb-5 items-start justify-center"}>
+          <TextInput
+            className={"ml-3"}
+            placeholder="*Enter Username"
+            placeholderTextColor="#000000"
+            onChangeText={(value) => setUsername(value)} />
```

Comment:
`(value) => setUsername(value)` is actually better as `setUsername`. You're creating an function that takes one parameter and calls `setUsername`, but `setUsername` is already a function that takes one argument and calls `setUsername`!

Reply:
i've tried different variations and the one I had originally is the only one not giving me an error.